src/crypto-providers/util.ts

```diff
@@ -199,37 +198,26 @@ const filterSigningFiles = (
   }
 }
 
-const hwSigningFileToPubKeyHash = (signingFile: HwSigningData): Uint8Array => (
-  cardanoSerialization.PublicKey.from_bytes(
-    splitXPubKeyCborHex(signingFile.cborXPubKeyHex).pubKey,
-  ).hash().to_bytes()
+const hwSigningFileToPubKey = (signingFile: HwSigningData): Buffer => (
+  splitXPubKeyCborHex(signingFile.cborXPubKeyHex).pubKey
 )
 
-const findPathForKeyHash = (
-  keyHash: Buffer, signingFiles: HwSigningData[],
-): BIP32Path | undefined => {
-  const signingFile = signingFiles.find((file) => keyHash.equals(hwSigningFileToPubKeyHash(file)))
-  return signingFile?.path
+const hwSigningFileToPubKeyHash = (signingFile: HwSigningData): Uint8Array => {
```

Comment:
Why are we using `Uint8Array` here and `Buffer` in most other places?

Reply:
Probably because cardano-serialization-lib's `to_bytes()` returns `Uint8Array` (btw, `Buffer` is a proper subclass of `Uint8Array`). I changed `hwSigningFileToPubKeyHash` to return `Buffer`.